docs/api-testing/environments.md

```diff
@@ -7,51 +7,63 @@ description: "Gain a wide range of options to mix and match your test settings w
 
-1. Visit the **Environment** section in the test project section:
-  <img src={useBaseUrl('img/api-fortress/2021/04/accessEnvironment.png')} alt="access environment"/>
+There are multiple ways to create and access Test environments:
 
-2. Add the details (typically the target domain) and select **Confirm**:
-  <img src={useBaseUrl('img/api-fortress/2021/04/createEnvironment.png')} alt="create environment"/>
+### From a Project
+1. From within a Project, go to the **Environments** section.<br/><img src={useBaseUrl('img/api-fortress/2021/04/accessEnvironment.png')} alt="access environment"/>
+2. Click **Create Environment**.
+3. Enter a name for your environment, then click **Save** when finished.<img src={useBaseUrl('img/api-fortress/2021/04/createEnvironment.png')} alt="create environment" width="300"/>
+4. From here, you can:
+   * Add variable(s) manually by clicking **New Entry** > Enter **Key** and **Value** pairs > **Confirm**.<br/><img src={useBaseUrl('img/api-fortress/2021/04/manualEntry.png')} alt="create environment" width="500"/>
+   * Import an existing environment by clicking **Import** > **Choose File** (accepted file types are Postman, .json, .csv).<br/><img src={useBaseUrl('img/api-fortress/2021/04/importEnv.png')} alt="create environment" width="500"/>
 
-3. Add the variable details and select **Confirm**:
-  <img src={useBaseUrl('img/api-fortress/2021/04/addDomainVar.png')} alt="variable in environment"/>
+ The resulting environment (along with its defined variables) will now appear each of your Tests as an option in the environment dropdown.
 
-  You can add multiple variables to each environment:
-  <img src={useBaseUrl('img/api-fortress/2021/04/result.png')} alt="result in environment"/>
+### From a Test
+1. Locate your desired Test from any of the following sections: **Tests**, **Compose**, **Schedule**, or **HTTP Client**. These are all of the places in the UI where you can run a Test.
+2. Under **Run Configuration**, click the **No environment** dropdown.
+3. Select **Add item**.
+4. Enter a name for your environment, then click **Confirm**.
+5. Click **Create variable**, enter **Key** and **Value** pairs, then click **Confirm**.
+6. When you're finished adding variables, click the **Close Environments Editor** icon.<br/><img src={useBaseUrl('img/api-fortress/2021/04/closeEditor.png')} alt="create environment" width="300"/>
 
-4. The resulting environment (along with the defined variables) is now selectable and appears in the environment section:
-  <img src={useBaseUrl('img/api-fortress/2021/04/result2.png')} alt="result in tests"/>
+This new environment will be selected by default in the dropdown.
 
-By activating a preset in the tests section, you will be able to hit a different domain in the current session without actually changing the tests.
+## Using Environments
+Once you've added created an environment and added the variables you'll need, you can then run your Test against that environment.
```

Comment:
@kimsauce "Once you've added created an environment" added or created? :)

Reply:
yikes!